app/Repositories/Team/TeamRepository.php

```diff
@@ -203,6 +285,51 @@ public function setTeamProjectUser($team, $projects, $users)
         }
     }
 
+        /**
+     * Update Team / Project / User relationship
+     *
+     * @param $team
+     * @param $projects
+     * @param $users
+     */
+    public function updateTeamProjectUser($team, $projects, $users)
```

Comment:
Formatting issue.

Reply:
Corrected the formatting issue on TeamRepository@updateTeamProjectUser().